spec/fpm/package/python_spec.rb

```diff
@@ -164,24 +199,17 @@ def python_usable?
 
   context "python_scripts_executable is set" do
     it "should have scripts with a custom hashbang line" do
-      #subject.attributes[:python_install_bin] = '/usr/bin'
       subject.attributes[:python_scripts_executable] = "fancypants"
       subject.input("django")
 
-      # Get the default scripts install directory and use it to find django-admin.py from Django
-      # Then let's make sure the scripts executable setting worked!
-      if /darwin/ =~ RUBY_PLATFORM
-        # OSX
-        django_bindir = %x{python -c 'from distutils.sysconfig import get_config_var; print(get_config_var("BINDIR"))'}.chomp
-      elsif /cygwin|mswin|mingw|bccwin|wince|emx/ =~ RUBY_PLATFORM
-        # Windows
-        # (ptomulik) I'm not sure about this
-        django_bindir = %x{python -c 'from distutils.sysconfig import get_config_var; print(get_config_var("BINDIR"))'}.chomp
-      else
-        # Unix
-        #django_bindir = '/usr/local/bin'
-        django_bindir = %x{python -c 'from distutils.sysconfig import get_config_var; print(get_config_var("BINDIR"))'}.chomp
+      django_bindir = nil
+      # Determine, where 'easy_install' is going to install scripts
+      execmd('python', code_easy_install_default_script_dir) do |code,out,err|
```

Comment:
As an alternative, you could include this python script in fpm similar to the way the python package support already does this --

[python code](https://github.com/jordansissel/fpm/tree/master/lib/fpm/package/pyfpm) and [how it's invoked in fpm](https://github.com/jordansissel/fpm/blob/master/lib/fpm/package/python.rb#L181-L182).

Since this is just for a test, the python code could live in a file in spec/fixtures/ somewhere (common in other similar ruby projects) and you can invoke `safesystem("python spec/fixtures/default_script_dir.py")` possibly capturing the output to a file like done [here](https://github.com/jordansissel/fpm/blob/master/lib/fpm/package/python.rb#L182)

Thoughts?